x-pack/legacy/plugins/apm/cypress/cypress/integration/apm.feature

```diff
@@ -0,0 +1,9 @@
+Feature: APM
+
+  Background:
+    Given a user browses the APM UI application
+    When the user inspects the opbeans-go service
+
+  Scenario: Transaction duration charts
+    Then should redirect to correct path with correct params
```

Comment:
The `Then` clause is the outcome or the asserts to be done in the scenario. Ideally only one `Then` should be added per scenario